### ISSUE TITLE
agent-group chart fails schema validation

### ISSUE THREAD

**Issue by ctoestreich** [jitterbit/charts#13]
When using 

```yaml
dependencies:
  - name: agent-group
    version: 1.0.6
    repository: https://jitterbit.github.io/charts
```

The following error keeps getting spit out when trying to set params in values file

```bash
Error: values don't meet the specifications of the schema(s) in the following chart(s):
agent-group:
- (root): Additional property global is not allowed
```

I had to unzip archive, add this for schema, zip up and now reference a local file:/ instead of url to get it to work

```json
{
  "$schema": "http://json-schema.org/draft-07/schema#",
  "title": "Agent Group Chart Configuration",
  "description": "agent-group chart configuration",
  "definitions": {
    "image": {
      "description": "Docker image configuration",
      "type": "object",
      "additionalProperties": true,
      "required": [
        "repository",
        "pullPolicy"
      ],
      "anyOf": [
        {
          "required": [
            "tag"
          ]
        },
        {
          "required": [
            "digest"
          ]
        }
      ],
      "properties": {
        "repository": {
          "description": "Docker repository",
          "type": "string",
          "minLength": 1,
          "maxLength": 256
        },
        "tag": {
          "description": "Docker image tag",
          "type": [
            "string",
            "null"
          ],
          "pattern": "^[0-9a-zA-Z._-]+$",
          "minLength": 1,
          "maxLength": 256
        },
        "digest": {
          "description": "Docker image digest",
          "type": [
            "string",
            "null"
          ],
          "pattern": "^sha256:[0-9a-f]{64}$",
          "minLength": 71,
          "maxLength": 71
        },
        "pullPolicy": {
          "description": "Pull policy of Docker image",
          "type": "string",
          "enum": [
            "Always",
            "Never",
            "IfNotPresent"
          ]
        }
      }
    },
    "hpa": {
      "description": "Horizontal pod autoscaler configuration",
      "type": "object",
      "additionalProperties": false,
      "required": [
        "enabled",
        "minReplicas",
        "maxReplicas",
        "metrics"
      ],
      "properties": {
        "enabled": {
          "description": "Enable horizontal pod autoscaling",
          "type": "boolean"
        },
        "minReplicas": {
          "description": "Minimum number of replicas",
          "type": "integer",
          "minimum": 1
        },
        "maxReplicas": {
          "description": "Maximum number of replicas",
          "type": "integer",
          "minimum": 1
        },
        "metrics": {
          "description": "List of metrics to scale on",
          "type": "array",
          "additionalItems": false,
          "minItems": 0,
          "uniqueItems": true,
          "items": {
            "$ref": "https://raw.githubusercontent.com/instrumenta/kubernetes-json-schema/master/v1.12.8-standalone-strict/_definitions.json#/definitions/io.k8s.api.autoscaling.v2beta2.MetricSpec"
          }
        }
      }
    },
    "replicas": {
      "description": "Number of replicas set if the horizontal pod autoscaler is disabled or malfunctioning",
      "type": "integer",
      "minimum": 1
    },
    "resources": {
      "description": "Upper- and lower-bounded resource constraints of the containers",
      "$ref": "https://raw.githubusercontent.com/instrumenta/kubernetes-json-schema/master/v1.12.8-standalone-strict/_definitions.json#/definitions/io.k8s.api.core.v1.ResourceRequirements"
    },
    "revisionHistoryLimit": {
      "description": "Number of old replica sets to store in case of the need to rollback",
      "type": "integer",
      "minimum": 0
    },
    "strategy": {
      "description": "Update strategy",
      "$ref": "https://raw.githubusercontent.com/instrumenta/kubernetes-json-schema/master/v1.12.8-standalone-strict/_definitions.json#/definitions/io.k8s.api.apps.v1.DeploymentStrategy"
    },
    "affinity": {
      "description": "Pod affinity and anti-affinity definitions",
      "$ref": "https://raw.githubusercontent.com/instrumenta/kubernetes-json-schema/master/v1.12.8-standalone-strict/_definitions.json#/definitions/io.k8s.api.core.v1.Affinity"
    },
    "probe": {
      "description": "Container's readiness and liveness probe definition",
      "$ref": "https://raw.githubusercontent.com/instrumenta/kubernetes-json-schema/master/v1.12.8-standalone-strict/_definitions.json#/definitions/io.k8s.api.core.v1.Probe"
    },
    "env": {
      "description": "List of additional environment variables that may be specified in the container",
      "type": "array",
      "additionalItems": false,
      "minItems": 0,
      "uniqueItems": true,
      "items": {
        "$ref": "https://raw.githubusercontent.com/instrumenta/kubernetes-json-schema/master/v1.12.8-standalone-strict/_definitions.json#/definitions/io.k8s.api.core.v1.EnvVar"
      }
    }
  },
  "type": "object",
  "additionalProperties": false,
  "required": [
    "image",
    "agent",
    "hpa",
    "replicas",
    "resources",
    "revisionHistoryLimit",
    "affinity",
    "readinessProbe",
    "livenessProbe",
    "env"
  ],
  "properties": {

    "global": {
      "description": "Global",
      "type": "object"
    },
    "metadata": {
      "description": "Chart metadata",
      "type": "object"
    },
    "nameOverride": {
      "description": "Override the name of resources",
      "type": "string"
    },
    "image": {
      "description": "Docker image configuration",
      "$ref": "#/definitions/image"
    },
    "agent": {
      "description": "Agent Group specific configuration",
      "type": "object",
      "additionalProperties": false,
      "required": [
        "registerjson"
      ],
      "properties": {
        "registerjson": {
          "description": "agent group's register.json file",
          "oneOf": [
            {
              "type": "object",
              "additionalProperties": false
            },
            {
              "type": "object",
              "additionalProperties": false,
              "required": [
                "cloudUrl",
                "username",
                "password",
                "agentGroupId"
              ],
              "properties": {
                "cloudUrl": {
                  "description": "Base URL of the Jitterbit Harmony Cloud",
                  "type": "string"
                },
                "username": {
                  "description": "Encrypted username of the Agent Installer user (encrypted using JitterbitUtils)",
                  "type": "string"
                },
                "password": {
                  "description": "Encrypted password of the Agent Installer user (encrypted using JitterbitUtils)",
                  "type": "string"
                },
                "agentGroupId": {
                  "description": "Agent group ID from Management Console (hover over an agent group to see its ID)",
                  "type": "number"
                },
                "agentNamePrefix": {
                  "description": "Prefix for each auto-registered agent's name",
                  "type": "string"
                },
                "deregisterAgentOnDrainstop": {
                  "description": "Performs agent deregistration on drainstop/JVM shutdown (defaults to false)",
                  "type": "boolean"
                },
                "retryCount": {
                  "description": "Number of retries if agent is having issues making the call to Harmony cloud for registration (defaults to 10, valid range is 0-300)",
                  "type": "integer",
                  "minimum": 0,
                  "maximum": 300
                },
                "retryIntervalSeconds": {
                  "description": "Number of seconds the agent will wait before retrying. This number doubles every retry to a maximum of 600 seconds (10 minutes). Defaults to 5, valid range 5-600.",
                  "type": "integer",
                  "minimum": 5,
                  "maximum": 600
                }
              }
            },
            {
              "type": "null"
            }
          ]
        },
        "jitterbitconf": {
          "description": "Agent group's jitterbit.conf file",
          "type": [
            "string",
            "null"
          ]
        }
      }
    },
    "hpa": {
      "description": "Horizontal pod autoscaler configuration",
      "$ref": "#/definitions/hpa"
    },
    "replicas": {
      "description": "Number of replicas set if the horizontal pod autoscaler is disabled or malfunctioning",
      "$ref": "#/definitions/replicas"
    },
    "resources": {
      "description": "Upper- and lower-bounded resource constraints of the containers",
      "$ref": "#/definitions/resources"
    },
    "revisionHistoryLimit": {
      "description": "Number of old replica sets to store in case of the need to rollback",
      "$ref": "#/definitions/revisionHistoryLimit"
    },
    "affinity": {
      "description": "Pod affinity and anti-affinity definitions",
      "$ref": "#/definitions/affinity"
    },
    "readinessProbe": {
      "description": "Periodic probe of container service readiness. Container will be removed from service endpoints if the probe fails. More info: https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle#container-probes\n",
      "$ref": "#/definitions/probe"
    },
    "livenessProbe": {
      "description": "Periodic probe of container liveness. Container will be restarted if the probe fails. More info: https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle#container-probes\n",
      "$ref": "#/definitions/probe"
    },
    "env": {
      "description": "List of additional environment variables that may be specified in the container",
      "$ref": "#/definitions/env"
    }
  }
}

```

**Comment by ctoestreich** [comment]
@jb-wau I am happy to fix this, but is anyone watching this repo?  The 1.0.7 charts were never actually released just built and a github release created.  Doesn't seem that they were actually released to helm.

**Comment by jgomez-jb** [comment]
@ctoestreich sorry for the late response, and thanks for the observations in regards to `1.0.7`, we are looking into it.
Our chart currently does not support the `global` object, I will review it with the team to include it and avoid the workaround. Thanks.

**Comment by jgomez-jb** [comment]
This has been addressed at `1.0.8`
